src/server/physics/world.js

```diff
@@ -1,149 +1,230 @@
 module.exports = function(Ammo, config) {
-	// Physics letiables
-	let collisionConfiguration,
-		dispatcher,
-		broadphase,
-		solver,
+	let _collisionConfiguration,
+		_dispatcher,
+		_broadphase,
+		_solver,
 		physicsWorld;
```

Comment:
~~Should be prefixed with an underscore.~~ Nevermind, just realised it's a sort of forward declaration. It might be a good idea to add a comment for that.